samples/test/util.py

```diff
@@ -204,16 +203,32 @@ class KfpArtifact:
     name: str
     uri: str
     type: str
+    custom_properties: dict
 
     @classmethod
     def new(
-        cls, mlmd_artifact: metadata_store_pb2.Artifact,
-        mlmd_artifact_type: metadata_store_pb2.ArtifactType
+            cls, mlmd_artifact: metadata_store_pb2.Artifact,
+            mlmd_artifact_type: metadata_store_pb2.ArtifactType
     ):
+        custom_properties = {}
+        for k, v in mlmd_artifact.custom_properties.items():
+            raw_value = None
+            if v.string_value:
+                raw_value = v.string_value
+            if v.int_value:
+                raw_value = v.int_value
```

Comment:
I believe there is also float_value

Reply:
There is no float value
Error logs:
`  File "/workdir/samples/test/util.py", line 220, in new
    if v.float_value:
AttributeError: float_value`

Reply:
Same for paratemer type https://github.com/kubeflow/pipelines/blob/b46f1b86f18a7539985ed15c0bb727281eb4d353/samples/test/util.py#L373

Reply:
It's called `double_value` (see v0.25.1 protos for example: https://github.com/google/ml-metadata/blob/v0.25.1/ml_metadata/proto/metadata_store.proto#L24, but it's been there since the beginning [or first public release v013.2]) and there can be more in even more recent versions of MLMD.

Maybe this is missing from multiple places?

Reply:
I think it's missing in multiple places. I will create a issue to track the fix of it.

Reply:
https://github.com/kubeflow/pipelines/pull/5591 submitted